common/FormattingUtils.js

```diff
@@ -74,16 +73,29 @@ function createShortHexString(hex) {
 
 // Take in either a transaction or an account and generate an etherscan link for the corresponding
 // network formatted in markdown.
-function createEtherscanLinkMarkdown(web3, hex) {
+function createEtherscanLinkMarkdown(hex, web3 = null) {
```

Comment:
Not overly happy with this implementation but I don't want to make downstream sync. Please let me know if you see a cleaner way to deal with this.

Reply:
One option is to remove the dependency on web3 in its entirety and feed in the `networkID` into `createEtherscanLinkMarkdown` as a param. This would be great but it would mean that every time a function calls this it would need to know the network ID. As this is never stored anywhere it would mean putting this async function call in the caller, every time. 

This could be ok if we bubble up where this call happens and then store it within the clients. For example, we could put this within the `ExpiringMultiPartyClient` and call it on the first boot.